sdk/purview/purview-workflow-rest/swagger/README.md

```diff
@@ -0,0 +1,24 @@
+# Azure Purview Workflow TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: "@azure-rest/purview-workflow"
+title: Purview Workflow
+description: Purview Workflow Client
+generate-metadata: false
+generate-test: false
+generate-sample: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/purviewWorkflow.json
```

Comment:
Please replace `main` for the specific commit you are using. This will help in the future if you need to generate against the same version